src/database/Connector.java

```diff
@@ -65,7 +65,7 @@ public Connector() {
    * @param password
    * @return true if the insertion is successful and complete; false, if the insertion fails
    */
-  public boolean addAccount(String username, String password) {
+  public synchronized boolean addAccount(String username, String password) {
 
     String sqlInsertAccount = "insert into "+tableName + "(username, password)" + "values(?,?)";
```

Comment:
Space between binary operators and after commas, it's good practice to capitalize SQL commands
`String sqlInsertAccount = "INSERT INTO " + tableName + "(username, password)" + "values(?, ?)";`